### DISCUSSION TITLE
Punishment list

### DISCUSSION THREAD

**Discussion by jurajplayercze** [Zrips/CMI#5857] [Enhancement]
I would appreciate option for showing all punishments for player. What that means? 

Good example is, if you want to "hire" helper for your server and you want to see if he was anytime punished for something, but you don't have any command(and it looks like, you don't have database for it too)  for that.

Command like /punishmentlist [nickname] (or anything you want) would write in chat whole history of [nickname] his punishments, like warnings, expired bans, kicks, etc. Also option in config to disable that feature and disable saving the informations about every punishment for owners that doesn't want to have little bigger database(if it's big server)

Visit https://github.com/Zrips/CMI/discussions/new to post new idea or suggestion
Posting new suggestion in ticket section could result in their removal.

**Comment by mrfloris** [comment]
> Visit https://github.com/Zrips/CMI/discussions/new to post new idea or suggestion
> Posting new suggestion in ticket section could result in their removal.

This is probably the best. 

But I agree, a history of set bans would be nice to have in the future, the same for warnings.

**Comment by CloudeLecaw** [comment]
We let our ban and warnings currently get post ti a discord special channel that only higher staff members are allowed to few. Well i know its not the best way specialy when the list become longer. But currently our only sensfull Option
